Unknown variable in preprocessor statement

When trying to compile with IL2CPP enabled in Unity, the compiler throws the following error:

"Assets\SQLiteForUnity\Runtime\SQLite-net\src\SQLite.cs(2615,26): error CS0103: The name 'prop' does not exist in the current context"

After investigating the referenced line it became obvious, that the "prop" variable in line 2615 in the preprocessor statement does not get declared anywhere. Maybe due to some prior refactoring.

https://github.com/praeclarum/sqlite-net/blob/758ad7a5f5c93adbf83c7f0ce6d80dd9bf7be8ec/src/SQLite.cs#L2608-L2621

--- a/src/SQLite.cs
+++ b/src/SQLite.cs
@@ -2612,8 +2612,8 @@ namespace SQLite
 
 				var colAttr = member.CustomAttributes.FirstOrDefault (x => x.AttributeType == typeof (ColumnAttribute));
 #if ENABLE_IL2CPP
-                var ca = prop.GetCustomAttribute(typeof(ColumnAttribute)) as ColumnAttribute;
-				Name = ca == null ? prop.Name : ca.Name;
+                var ca = member.GetCustomAttribute(typeof(ColumnAttribute)) as ColumnAttribute;
+				Name = ca == null ? member.Name : ca.Name;
 #else
 				Name = (colAttr != null && colAttr.ConstructorArguments.Count > 0) ?
 						colAttr.ConstructorArguments[0].Value?.ToString () :

